src/components/checkbox/Checkbox.tsx

```diff
@@ -58,7 +64,6 @@ const CheckboxCommon = css`
 
 const HiddenCheckbox = styled.input.attrs({ type: 'checkbox' })`
   ${CheckboxCommon}
-  visibility: hidden;
```

Comment:
Skeptical to this one, the whole point here is having the actual input element hidden so browser styling doesn't mess everything up. I googled "react checkbox" and the [first result](https://medium.com/@colebemis/building-a-checkbox-component-with-react-and-styled-components-8d3aa1d826dd) gave me something similar, so I'm guessing my approach isn't too uncommon.
Hidden items still take up space and are maybe interactable (can't recall), while `display: none` would just remove the element from the DOM obviously

Reply:
We can use `opacity: 0`, but not `visibility: hidden` as it, in my case, did not make the element interactable with tabs. Not exactly sure why, but I cannot find any examples using `visibility: hidden`. There are a few examples using `opacity: 0`, and that works, but that will also hide our focus outline.

Everything seems to be working correctly when just hiding the checkbox behind the fake checkbox, not doing anything with the visibility/opacity. The example from the link does hide the checkbox, but it does that using `clip: rect(0 0 0 0);`. I have never seen that approach before but could be that it is even more robust.